src/game_api/entity.cpp

```diff
@@ -838,3 +838,14 @@ uint32_t Movable::get_behavior()
     }
     return 0; // there is no id 0, but i can be wrong
 }
+
+void Movable::set_gravity(float gravity)

```

Comment:
Is this a stupid way to do this? Does everything blow up if you do this every frame for example?